indicator_creation_rules/new_account_logging.yml

```
@@ -0,0 +1,97 @@
+# Monitors for account creation and adds an entry to the KVStore for the user. This depends on the
+# event_type of ACCOUNT_CREATED to be in the data model for the log source and will work in tandem
+# with a helper function that checks for the userid in the KV store.
+
+AnalysisType: rule
+Filename: new_account_logging.py
+RuleID: Standard.NewAccountCreated
+DisplayName: New User Account Created
+Enabled: true
+LogTypes:
+  - OneLogin.Events
+  - AWS.CloudTrail
+Tags:
+  - DataModel
+  - IndicatorCollection
```

Tags can have spaces, this could be:
```suggestion
  - Indicator Collection
```
If you think that would improve readability.

---

Nice, I do think that would be more readable. Will change.